### ISSUE TITLE
Tenant timeout loading

### ISSUE THREAD

**Issue by alarido** [digitalocean/netbox#419]
I have a big tenant thousands of IP addresses, hundreds of prefixes, vrfs, connections, devices, ... when I go to the tenant view it timeouts:

https://netbox.ae.local/tenancy/tenants/MYBIGTENANT/

I have been increasing timeout from default to several minutes, but it does not load anyways, it always timeouts, database and netbox server load is low:

Apache proxy error:
```
Proxy Error

The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request GET /tenancy/tenants/MYBIGTENANT/.

Reason: Error reading from remote server
```
supervisord netbox log error:

`2016-08-03 09:28:33 [37301] [ERROR] WORKER TIMEOUT (pid:37306)`

**Comment by jeremystretch** [comment]
@alarido I suspect that the way we're annotating stats for the tenant in a single query is causing a database performance issue. I've rewritten the view to use a simple, discrete `COUNT()` for each statistic now. Could you try pulling down the `develop` branch and seeing if it solves the issue for you?

**Comment by jeremystretch** [comment]
Should be fixed in v1.4.1 but please let me know if you still experience the problem.

**Comment by alarido** [comment]
It is fixed, thanks.
